extensions/wikia/GlobalNavigation/scripts/GlobalNavigationNotifications.js

```diff
@@ -1,3 +1,8 @@
+// TODO XW-1943
+// don't update this file as it's no longer being used
+// we plan to keep it only until the Design System Global Navigation proves to be stable on production
+// the logic was copied to extensions/wikia/DesignSystem/scripts/DesignSystemGlobalNavigationNotifications.js
+
 require(
 	['jquery', 'wikia.window', 'wikia.nirvana', 'wikia.delayedhover', 'wikia.globalNavigationDropdowns'],
 	function($, window, nirvana, delayedHover, dropdowns) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This function has 386 lines, which is greater than the 100 lines authorized. Split it into smaller functions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AS138)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Function has a complexity of 65 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AFunctionComplexity)